src/devdb/helpers.py

```diff
@@ -6,12 +6,15 @@
 
 def get_data(branch):
     rv = {}
-    url = f"https://edm-publishing.nyc3.digitaloceanspaces.com/db-developments/latest/output"
+    url = f"https://edm-publishing.nyc3.digitaloceanspaces.com/db-developments/518-Historic-QAQC/latest/output"
```

Comment:
How did this branch get created? Did you run the feature branch on your local?

Reply:
Uh good q - maybe in the process of testing your PR?

Reply:
Word, that makes sense. Good sign that it worked even when we weren't very careful lol